Cargo.lock

```diff
@@ -1149,13 +1164,26 @@ version = "2.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "5d85653f070353a16313d0046f173f70d1aadd5b42600a14de626f0dfb3473a5"
 dependencies = [
- "byteorder 1.3.4",
+ "byteorder",
  "digest 0.8.1",
  "rand_core 0.5.1",
  "subtle 2.2.3",
  "zeroize",
 ]
 
+[[package]]
```

Comment:
If we bump `ed25519-dalek` to `v1.0.1` in both [`primitives/core/Cargo.toml`](https://github.com/paritytech/substrate/blob/184527825c0f9c2037df99406fec029d3cfbba12/primitives/core/Cargo.toml#L44) and [libp2p-core/Cargo.toml](https://github.com/libp2p/rust-libp2p/blob/e5adb67d73b2168f6112df8308a923494d9b831c/core/Cargo.toml#L15)  we wouldn't need both `curve25519-dalek` `2.1.0` and `3.0.0` with the latter introduced through `libp2p-noise`.

Given that this requires a change within `rust-libp2p` this will have to wait for now.